General.md

```diff
@@ -23,3 +23,30 @@ function deprecateAshForm() {
 
 class Rectangle { //ES6 and Ember classes still use UpperCamelCase by general consensus
 ```
+
+## Formatiing
+> Why? All of our code should be formatted so that it is clean, concise, and easily readable by any developer on the team
+* For properties, put a space after the semicolon (`:`)
```

Comment:
The example should display a semicolon, as stated, rather than a colon. This was a typo that was carried over from the previous standard location.

Reply:
Will fix